src/components/Txt/index.tsx

```diff
@@ -3,7 +3,23 @@ import styled, { css } from 'styled-components';
 import { style } from 'styled-system';
 import asRendition from '../../asRendition';
 import { RenditionSystemProps, Theme } from '../../common-types';
-import { monospace } from '../../utils';
+import { px, monospace } from '../../utils';
+
+export const code = (props: ThemedTxtProps) =>
+	props.code
+		? css`
+				font-family: ${(props) => props.theme.monospace};
+				padding: 2px 4px;
+				font-size: 90%;
+				color: #c7254e;
+				background-color: #f9f2f4;
+				border-radius: 2px;
+				white-space: normal;
+				word-wrap: break-word;
+				font-size: 1em;
```

Comment:
Let's remove this, since it will not have any effect after we remove the `font-size: 90%;` from above.